lib/manageiq/ui/classic/engine.rb

```diff
@@ -1,5 +1,10 @@
 require 'rails/all'
+require 'sass-rails'
+require 'coffee-rails'
 require 'patternfly-sass'
+require 'lodash-rails'
+require 'angular-ui-bootstrap-rails'
+require 'jquery-hotkeys-rails'
```

Comment:
we need them constantly in development and also for asset precompilation in production